app/src/main/java/org/fossasia/openevent/general/attendees/AttendeeFragment.kt

```diff
@@ -767,8 +767,9 @@ class AttendeeFragment : Fragment(), ComplexBackPressFragment {
     }
 
     private fun sendToken() {
-        val card = Card(rootView.cardNumber.text.toString(), attendeeViewModel.monthSelectedPosition,
+        val builder = Card.Builder(rootView.cardNumber.text.toString(), attendeeViewModel.monthSelectedPosition,
             attendeeViewModel.yearSelectedPosition, rootView.cvc.text.toString())
+        val card = builder.build()
```

Comment:
Directly call build and remove obsolete builder variable

Reply:
Updated